stetho/src/main/java/com/facebook/stetho/common/android/FragmentCompatUtil.java

```diff
@@ -62,10 +78,12 @@ private static Object findFragmentForViewInActivity(
       FragmentCompat compat,
       Activity activity,
       View view) {
-    return findFragmentForViewInFragmentManager(
-        compat,
-        compat.forFragmentActivity().getFragmentManager(activity),
-        view);
+    Object fragmentManager = compat.forFragmentActivity().getFragmentManager(activity);
+    if (fragmentManager != null) {
```

Comment:
yeah. getFragmentManager()'s return type is @Nullable. Our implementation will never return null, but it used to when my code was in there that used reflection to peek at the field vs. calling the method which defer-creates the FragmentManager. We should fix that but I'm not sure of how going back to reflection here will affect performance (it's fairly low-pri, i.o.w.)